test/Microsoft.AspNetCore.Razor.Test.Common/Language/CodeGeneration/TestCodeRenderingContext.cs

```diff
@@ -15,7 +15,7 @@ public static class TestCodeRenderingContext
         {
             var codeWriter = new CodeWriter();
             var documentNode = new DocumentIntermediateNode();
-            var options = RazorCodeGenerationOptions.CreateDesignTimeDefault();
+            var options = RazorCodeGenerationOptions.CreateDesignTime(c => { });
```

Comment:
> As a side note, for lambdas where you don't use the variable it's common practice to use the _ character

This is absolutely not the case in C# and has never been part of our teams' guidelines. `_` is now a reserved word in some contexts, we shouldn't sprinkle it all over